packages/material-ui/src/styles/index.d.ts

```diff
@@ -58,3 +58,4 @@ export {
 export { ComponentsProps } from './props';
 export { ComponentsVariants } from './variants';
 export { ComponentsOverrides } from './overrides';
+export { StyledEngineProvider } from '@material-ui/styled-engine';
```

Comment:
The alternative import path:

```suggestion
export StyledEngineProvider from '@material-ui/styled-engine/StyledEngineProvider';
```

Reply:
This did not worked - https://app.circleci.com/pipelines/github/mui-org/material-ui/42016/workflows/86bb27a2-0a08-4888-8b71-2ebbfe4655f5/jobs/251454

Reply:
https://github.com/mui-org/material-ui/blob/205258b56faf4ac8096d10b62e22458dab52fb8e/packages/material-ui/scripts/rollup.config.js#L10

Reply:
Interesting, it's failing 4 steps in the CI on the same thing

Reply:
Let me try to update the `rollup.config.js` and see if it will work

Reply:
Looks like this worked.